libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/ConfigurationBotFrameworkAuthentication.cs

```diff
@@ -36,8 +36,8 @@ public ConfigurationBotFrameworkAuthentication(IConfiguration configuration, Ser
             var toChannelFromBotOAuthScope = configuration.GetSection("ToChannelFromBotOAuthScope")?.Value;
             var toBotFromChannelTokenIssuer = configuration.GetSection("ToBotFromChannelTokenIssuer")?.Value;
             var oAuthUrl = configuration.GetSection("OAuthUrl")?.Value;
-            var toBotFromChannelOpenIdMetadataUrl = configuration.GetSection("ToBotFromChannelOpenIdMetadataUrl")?.Value;
-            var toBotFromEmulatorOpenIdMetadataUrl = configuration.GetSection("ToBotFromEmulatorOpenIdMetadataUrl")?.Value;
+            var toBotFromChannelOpenIdMetadataUrl = configuration.GetSection(AuthenticationConstants.BotOpenIdMetadataKey)?.Value;
+            var toBotFromEmulatorOpenIdMetadataUrl = configuration.GetSection(AuthenticationConstants.BotOpenIdMetadataKey)?.Value;
```

Comment:
The key for OpenId metadata used by bot authors today is `AuthenticationConstants.BotOpenIdMetadataKey`. So, I changed this to match the currently used key. @johnataylor, please let me know if this is ok.